conformance-suites/2.0.0/conformance2/textures/misc/copy-texture-image.html

```diff
@@ -80,8 +80,12 @@
         "RG8",
         "RGB8",
         "RGBA8",
+        "R8_SNORM",
```

Comment:
Originally this test is written against the list of valid internalformats from https://www.khronos.org/opengles/sdk/docs/man3/html/glCopyTexImage2D.xhtml

I understand the man page isn't the spec, but I am curious why the man page so confidently offers a list of the valid internalformats?